README.md

```diff
@@ -25,9 +25,19 @@ This will ensure that the `django-admin` command is available in your shell. Fro
 you can create a new project with the following command:
 
 ```shell
-django-admin startproject --template path/to/django-template/template/ --extension py,env,sh,toml,yml <project_name>
+django-admin startproject --template path/to/django-template/template/ --extension py,env,sh,toml,yml --exclude .venv <project_name>
 ```
 
+> [!NOTE]
+> When starting a new Django project using a custom template, some directories and files
+> that begin with a dot (.) are not automatically included. This exclusion typically affects
+> important folders like `.github`, which is commonly used for GitHub Actions CI/CD workflows.
+> However, from Django 4.0, you can ensure that directories like `.github` are properly
+> copied to your new project. You can achieve this by using the `--exclude` option during
+> project initialization. If you omit this option, directories beginning with a dot, including
+> `.github/workflows`, will be left out.
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏How does adding an exclusion mean that extra things get included?

I think I understand the problem, that Django's `startproject` command will ignore dot folders and files by default. But I am not sure I understand how adding a `--exclude` flag to make sure the `.venv` is ignored helps?

Aside: You shouldn't have a .venv in the template directory. So the exclude should not be necessary.

Reply:
Thank you @a-musing-moose for the review. That's a good question. 

I agree that using `--exclude .venv` here seems unusual. The issue with copying the GitHub workflows lies in Django's default behaviour, which excludes all folders starting with a `.`. I've considered a few options to address this:

- Manual Copy: We could add notes to the README documentation and have users manually copy the GitHub actions from the template to their new project.
- Script Automation: Another option is to create scripts that run the `django-admin startproject` command followed by the `cp` command to copy the necessary files.
- Custom Exclude: Alternatively, passing `--exclude` here overrides the default behaviour, excluding only the directories specified. For example, using `--exclude <FICTIONAL DIRECTORY>` would prevent Django from excluding the `.github` folder. Currently, there are no folders in the template that we want to exclude, so this approach may seem unnecessary. However it actually lets the github workflows to be copied, and I believe it still offers a more coherent solution compared to the previous options.

I'm open to the suggestion though.

Reply:
I updated the README doc to clarify this a bit more: [ccd6de7](https://github.com/ackama/django-template/pull/41/commits/ccd6de7dd183ad22e431210dc0e723549010f3d8)